tests/framework/CMakeLists.txt

```diff
@@ -96,7 +95,7 @@ add_dependencies (testing_framework_util generate_framework_config)
 add_library(testing_dependencies STATIC test_environment.cpp test_environment.h)
 target_link_libraries(testing_dependencies
     PUBLIC gtest Vulkan::Headers testing_framework_util shim-library)
-target_include_directories(testing_dependencies PUBLIC ${CMAKE_BINARY_DIR}/tests/framework)
```

Comment:
This was intentional, because the is a header file being generated at that path.

Reply:
`CMAKE_CURRENT_BINARY_DIR` is equivalent to `${CMAKE_BINARY_DIR}/tests/framework`.

Reply:
Oh, I thought this was happening inside a function. My bad, you are right about that.